uds/diagnostic_configuration/state_definitions.py

```
@@ -1,28 +1,45 @@
-"""Definitions of typical diagnostic communication states."""
+"""Predefined State objects representing common :ref:`diagnostic communication states <knowledge-base-states>`."""
 
 __all__ = ["DEFAULT_DIAGNOSTIC_SESSION_STATE",
            "DEFAULT_SECURITY_ACCESS_STATE",
            "DEFAULT_AUTHENTICATION_STATE",
-           "DEFAULT_SECURED_TRANSMISSION_STATE",
+           "DEFAULT_IGNITION_STATE",
            "DEFAULT_ENGINE_STATE",
+           "DEFAULT_SECURED_TRANSMISSION_STATE",
            "DEFAULT_ADDRESSING_TYPE_STATE"]
```

Ruff (0.15.20)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)